commands/riolu.js

```diff
@@ -17,15 +17,16 @@ module.exports = {
 	usage: "\n",
 
 	// This function is used to get a random image from Google and then send it to Discord
-	execute(message, args, client) {
+	execute(message) {
 		try {
 			const results = googleImages.search("Riolu"); // Indexing Google images
 			const reply = !results
 				? "No results"
 				: new Attachment(results[Math.floor(Math.random() * results.length)].url); // Randomize output
 			message.channel.send(reply);
 		} catch (e) {
-			return e;
+			console.error(e);
+			message.channel.send("Error happened, see the console"); // Error Handling
```

Comment:
There's already error catchingat [line 104 of the main js file](https://github.com/Matojeje/mato-bot/blob/master/testingBot.js#L104). Example from development:

![obrazek](https://user-images.githubusercontent.com/3956863/52671951-b4c65f80-2f1c-11e9-8f2d-f9eaf39fcc72.png)

Reply:
Removed the error handling in `riolu.js`, I did not see that.